modules/video/admin.js

```diff
@@ -0,0 +1,39 @@
+(function($) {
+
+  $(".snowball-main").on("open", ".snowball-block-video", function() {
+    var videoUrl = $(this).find(".video-url").val();
+    var videoID = parseVideoURL(videoUrl);
+
+    $(this).find(".video-id").val(videoID);
+    $(this).find(".video-id").trigger("change");
```

Comment:
There's a block-level event called `render` that you can use instead.

`$(this).trigger("render");`

Also applies to lines 17 and 30?